app/lib/sidebar/local_widgets/list_tile.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:app/navbar/navigation.dart';
+import 'package:flutter/material.dart';
+
+class CustomTile extends StatelessWidget {
```

Comment:
I wonder if we could have a better name than `CustomTile`. For example, your names `HomePage` `InProgressPage` `TravelHistory` are good names because I know what they represent right away. 

- [ ] What is a tile, and could we call this something like `QuickLinkTile`?

Reply:
A tile is the links inside the drawer (e.g. Travel history). I'll looking into proper flutter naming convention in the future.  Since this my own implementation of ListTile widget, I decided to name it 'CustomTile'.  This is an example of extracting whatever is inside the child.